components/footer/index.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright (c) IBM, Corp. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LitElement, html, type TemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { mediumIcon } from './icons/medium.js';
+import { qiskitLogoIcon } from './icons/qiskit-logo.js';
+import { slackIcon } from './icons/slack.js';
+import { twitterIcon } from './icons/twitter.js';
+import { youtubeIcon } from './icons/youtube.js';
+import styles from './index.scss';
+
+interface Link {
+  label: string;
+  url: string;
+}
+
+interface Column {
+  title: string;
+  items: Array<Link>;
+}
+
+@customElement('qiskit-footer')
+export class QiskitFooter extends LitElement {
+  static styles = [styles];
+
+  @property({ attribute: false })
+  info = [{ title: '', items: [{ label: '', url: '' }] }];
+
+  create_list_sections(items: Array<Link>) {
+    const aux: Array<TemplateResult> = [];
+    items.forEach((link: Link) => {
+      aux.push(html` <li>
+        <a
+          class="link"
+          style="cursor: pointer;"
+          label="${link.label}"
+          href="${link.url}"
+        >
+          <span>${link.label}</span>
+        </a>
+      </li>`);
+    });
+    return aux;
```

Comment:
I think [`.map()`](https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Array/map) is a better option than `.forEach()`

Reply:
Done!